PowerEditor/src/ScitillaComponent/FindReplaceDlg.h

```diff
@@ -401,7 +401,7 @@ private :
 	void updateCombo(int comboID);
 	void fillFindHistory();
     void fillComboHistory(int id, const std::vector<generic_string> & strings);
-	int saveComboHistory(int id, int maxcount, std::vector<generic_string> & strings);
+	int saveComboHistory(int id, int maxcount, std::vector<generic_string> & strings, bool saveEmpty = false);

```

Comment:
OK, I see why it's optional. Please make filter (IDD_FINDINFILES_FILTERS_COMBO) the same behaviour (to true).
And remove the default value - let all calls explicit.